cardano-db-sync/src/Cardano/DbSync/Sync.hs

```diff
@@ -295,10 +291,9 @@ dbSyncProtocols trce env metricsSetters plugin queryVar runDBThreadFunction vers
     minVersion :: Network.NodeToClientVersion
     minVersion = Network.NodeToClientV_8
 
-logDbState :: SyncDataLayer -> Trace IO Text -> IO ()
-logDbState dataLayer trce = do
-    let getLatestBlock = sdlGetLatestBlock dataLayer
-    mblk <- getLatestBlock
+logDbState :: Trace IO Text -> IO ()
+logDbState trce = do
+    mblk <- getDbLatestBlockInfo
     case mblk of
       Nothing -> logInfo trce "Cardano.Db is empty"
       Just block ->
```

Comment:
```suggestion
      Just tip ->
```

The bindings in `logDbState` should change to match the type, it used to be a `Block` but now it's a `TipInfo`